### ISSUE TITLE
Script crash after join a meet

### ISSUE THREAD

**Issue by WIFIGUN** [TobiasPankner/Teams-Auto-Joiner#27]
**Describe the bug**
Script crashes in after Join now page

**Video**
[https://youtu.be/_O0fAz3XYAk](https://youtu.be/_O0fAz3XYAk)

RUN python auto_joiner.py
```
[WDM] - Current google-chrome version is 85.0.4183
[WDM] - Get LATEST driver version for 85.0.4183
[WDM] - Driver [C:\Users\WONGH\.wdm\drivers\chromedriver\win32\85.0.4183.87\chromedriver.exe] found in cache
Timeout waiting for element: .use-app-lnk
Waiting for correct page...

Found Teams and Channels:
3D
        一般
20-21 中三級現場教室
        一般
        中文科
        中史科
        化學科
        生物科
        生活與社會科
        地理科
        佛化科
        物理科
        英文科
        音樂科
        商業科
        視藝科
        電腦科
        數學科
        閱讀科
        歷史科
        體育科

Start [s], Reload teams [r], Quit [q]
s

[12:03:25] Updating channels
Wating for 30s
Joined meeting: 20-21 中三級現場教室 > 音樂科
Traceback (most recent call last):
  File "auto_joiner.py", line 442, in <module>
    main()
  File "auto_joiner.py", line 417, in main
    if join_newest_meeting(teams):
  File "auto_joiner.py", line 254, in join_newest_meeting
    browser.find_element_by_css_selector("span[data-tid='appBarText-Teams']").click()
  File "C:\Python38\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 598, in find_element_by_css_selector
    return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
  File "C:\Python38\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "C:\Python38\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python38\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"span[data-tid='appBarText-Teams']"}
  (Session info: chrome=85.0.4183.121)
```

**Configuration file**
```
{
  "email": "",
  "password": "",

  "run_at_time": "",

  "start_automatically": false,
  "organisation_num" : 1,
  "random_delay": true,
  "check_interval": 10,

  "auto_leave_after_min": -3,
  "leave_if_last": false,

  "headless" : false,
  "mute_audio": true,
  "chrome_type": "google-chrome",

  "blacklist": [
    {
      "team_name": "",
      "channel_names": [
        "",
        ""
      ]
    },


     {
      "team_name": "",
      "channel_names": [
      ]
    }


  ]
}
```

**Desktop**
 - OS: Windows10 64bit
 - Browser chrome
 - Version 85.0.4183.121

**Comment by TobiasPankner** [comment]
Thanks for the video, I think I know what the issue is and will fix it as soon as possible

**Comment by TobiasPankner** [comment]
Should be fixed with commit de34343, it was an issue with languages other than germnan or english (which I test).
